Extensions/PanelSpriteObject/panelspriteruntimeobject-pixi-renderer.js

```diff
@@ -40,12 +41,19 @@ gdjs.PanelSpriteRuntimeObjectPixiRenderer.prototype.getRendererObject = function
 
 gdjs.PanelSpriteRuntimeObjectPixiRenderer.prototype.ensureUpToDate = function() {
     if (this._spritesContainer.visible && this._wasRendered) {
-        this._spritesContainer.cacheAsBitmap = true;
+        this._spritesContainer.alpha = this._opacity;
```

Comment:
Can we add a comment here explaining why it's needed?
"// Update the alpha of the container to make sure that it's applied. If not done [...]"

The goal is to give the future reader/developer an explanation of what was wrong and how is this helping :)